src/java/com/unifina/signalpath/blockchain/WebsocketEthereumJsonRpc.java

```diff
@@ -0,0 +1,71 @@
+package com.unifina.signalpath.blockchain;
+
+import com.google.gson.JsonParser;
+import org.apache.log4j.Logger;
+import javax.websocket.*;
+import javax.websocket.Session;
+import java.io.IOException;
```

Comment:
Sort imports. You can do it automatically with IDEA:
- Preferences -> Editor -> Auto Import -> [x] Optimize imports on the fly